carina-aws-s3/src/main/java/com/qaprosoft/amazon/AmazonS3Manager.java

```diff
@@ -50,14 +59,16 @@ public static AmazonS3Manager getInstance() {
             synchronized (AmazonS3Manager.class) {
                 if (instance == null) {
                     instance = new AmazonS3Manager();
-                    String accessKey = Configuration.get(Parameter.ACCESS_KEY_ID);
-                    String secretKey = Configuration.get(Parameter.SECRET_KEY);
+                    CryptoTool cryptoTool = new CryptoTool(Configuration.get(Parameter.CRYPTO_KEY_PATH));
+                    Pattern CRYPTO_PATTERN = Pattern.compile(SpecialKeywords.CRYPT);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3AS00117)